File: rc-application/src/main/java/com/transkript/reportcard/business/service/i/ClassListService.java
package com.transkript.reportcard.business.service.i;

import com.transkript.reportcard.api.dto.request.ClassListRequest;
import com.transkript.reportcard.api.dto.response.ClassListResponse;
import kotlin.Metadata;
import org.jetbrains.annotations.Nullable;

public interface ClassListService {
    
    ClassListResponse getClassList( ClassListRequest var1);
}

File: rc-application/src/main/java/com/transkript/reportcard/business/service/i/ClassListService.kt
package com.transkript.reportcard.business.service.i

import com.transkript.reportcard.api.dto.request.ClassListRequest
import com.transkript.reportcard.api.dto.response.ClassListResponse

interface ClassListService {
    fun getClassList(request: ClassListRequest?): ClassListResponse?
}
